test/parallel/test-assert.js

```diff
@@ -256,68 +256,6 @@ a.throws(() => thrower(TypeError), (err) => {
 const circular = { y: 1 };
 circular.x = circular;
 
-function testAssertionMessage(actual, expected, msg) {
-  try {
-    assert.strictEqual(actual, '');
-  } catch (e) {
-    assert.strictEqual(
-      e.message,
-      msg || strictEqualMessageStart +
-             `+ actual - expected\n\n+ ${expected}\n- ''`
-    );
-    assert.ok(e.generatedMessage, 'Message not marked as generated');
-  }
-}
-
-function testShortAssertionMessage(actual, expected) {
-  testAssertionMessage(actual, expected, strictEqualMessageStart +
-                                         `\n${inspect(actual)} !== ''\n`);
-}
-
-testShortAssertionMessage(null, 'null');
-testShortAssertionMessage(true, 'true');
-testShortAssertionMessage(false, 'false');
-testShortAssertionMessage(100, '100');
-testShortAssertionMessage(NaN, 'NaN');
-testShortAssertionMessage(Infinity, 'Infinity');
-testShortAssertionMessage('', '""');
-testShortAssertionMessage('foo', '\'foo\'');
-testShortAssertionMessage(0, '0');
-testShortAssertionMessage(Symbol(), 'Symbol()');
-testAssertionMessage([], '[]');
-testAssertionMessage(/a/, '/a/');
-testAssertionMessage(/abc/gim, '/abc/gim');
-testAssertionMessage({}, '{}');
-testAssertionMessage(undefined, 'undefined');
-testAssertionMessage(-Infinity, '-Infinity');
-testAssertionMessage([1, 2, 3], '[\n+   1,\n+   2,\n+   3\n+ ]');
-testAssertionMessage(function f() {}, '[Function: f]');
-testAssertionMessage(function() {}, '[Function]');
-testAssertionMessage(circular, '{\n+   x: [Circular],\n+   y: 1\n+ }');
-testAssertionMessage({ a: undefined, b: null },
-                     '{\n+   a: undefined,\n+   b: null\n+ }');
-testAssertionMessage({ a: NaN, b: Infinity, c: -Infinity },
-                     '{\n+   a: NaN,\n+   b: Infinity,\n+   c: -Infinity\n+ }');
-
-// https://github.com/nodejs/node-v0.x-archive/issues/5292
-try {
-  assert.strictEqual(1, 2);
-} catch (e) {
-  assert.strictEqual(
-    e.message,
-    `${strictEqualMessageStart}\n1 !== 2\n`
-  );
-  assert.ok(e.generatedMessage, 'Message not marked as generated');
-}
-
-try {
-  assert.strictEqual(1, 2, 'oh no'); // eslint-disable-line no-restricted-syntax
-} catch (e) {
-  assert.strictEqual(e.message, 'oh no');
-  // Message should not be marked as generated.
-  assert.strictEqual(e.generatedMessage, false);
-}
-
```

Comment:
All of the tests above should be kept as they test code that has not been removed.